### PR TITLE
[WIP] chore(Carbon): update to v10.17.0

### LINKED ISSUE #1323: Update to Carbon v10.16.x
### Summary

This issue is to track the update to Carbon ~10.13.x~ and any associated items that we're waiting on being available upstream in Carbon.

**Additional context**
- [ ] The duplicated Carbon combobox component in CarbonCombobox.jsx from https://github.com/IBM/carbon-addons-iot-react/pull/1316 should be removed with this when updating to carbon-components-react@7.13.0
- [ ] We have additional changes being made in #1324 
- [ ] Review the testing console error exceptions - we might be able to remove some

### Available extra resources

The core team will likely make this upgrade.

### LINKED ISSUE #1469: [Dropdown] [A11Y] Accessability violations in Carbon Dropdown
### Describe the bug

Accessibility scanner is picking up violations in the dropdown component. We cannot fix these in the Graphite repo. 

### To Reproduce

1. Go to '.https://carbon-addons-iot-react.com/?path=/story/dropdown--default'
2. Click on 'Accessibility' tab
3. See error

Screen Shot 2020-08-06 at 12 19 24 PM

Relates to:
https://jsw.ibm.com/browse/GRAPHITE-15962
https://jsw.ibm.com/browse/GRAPHITE-15958

### PR DESCRIPTION
**Pull request by dakahn**
Closes #1323 
Closes #1469 

Smoke testing our v10.17 release candidate (with a bonus v10.16 bump coming along for the ride :surfer:)

### To test 
I'll include a list of bug fixes per relevant packages to check for bugs and breaks:
#### carbon-components-react
- fix(dropdown): add ...other prop (#6479) ([`810168bf8`](https://github.com/carbon-design-system/carbon/commit/810168bf8))
- fix(Copy): avoid label text mismatch (#6428) ([`177a6bd77`](https://github.com/carbon-design-system/carbon/commit/177a6bd77))
- fix(SideNav): a11y fixes for SideNavMenu, MenuItem (#6468) ([`efa7a9bb1`](https://github.com/carbon-design-system/carbon/commit/efa7a9bb1))
- fix(dropdown,multiselect): add support for refs on buttons (#6441) ([`2485371ed`](https://github.com/carbon-design-system/carbon/commit/2485371ed))
- fix(react): update downshift to 5.2.1 (#6418) ([`69151cc8c`](https://github.com/carbon-design-system/carbon/commit/69151cc8c))
- fix(multiselect): remove aria-selected prop (#6440) ([`2692813cc`](https://github.com/carbon-design-system/carbon/commit/2692813cc))
- fix(pagination-nav): add data-page attributes (#6437) ([`8c84f349b`](https://github.com/carbon-design-system/carbon/commit/8c84f349b))
- fix(react): add prop adapter for downshift changes (#6427) ([`0037736f7`](https://github.com/carbon-design-system/carbon/commit/0037736f7))
- fix(react): update useId to match id on server and client (#6369) ([`6d3e3dc6c`](https://github.com/carbon-design-system/carbon/commit/6d3e3dc6c))
- fix(data-table): sortable, boolean column, and overflow menu row fixes (#6268) ([`5d7986990`](https://github.com/carbon-design-system/carbon/commit/5d7986990))
- feat(CodeSnippet): allow copy button to be optional (#6505) ([`ad4d9a298`](https://github.com/carbon-design-system/carbon/commit/ad4d9a298))
- feat(grid): add narrow grid mode (#6520) ([`2c7f9aab5`](https://github.com/carbon-design-system/carbon/commit/2c7f9aab5))
- feat(modal): support alertdialog aria role (#5955) ([`825f03ab7`](https://github.com/carbon-design-system/carbon/commit/825f03ab7))
- feat(text-input): add warn prop (#5918) ([`a5e27b74c`](https://github.com/carbon-design-system/carbon/commit/a5e27b74c))

#### carbon-components
- fix(text-area): apply disabled text token to textarea content (#6495) ([`b8ff37f69`](https://github.com/carbon-design-system/carbon/commit/b8ff37f69))
- fix(Copy): avoid label text mismatch (#6428) ([`177a6bd77`](https://github.com/carbon-design-system/carbon/commit/177a6bd77))
- fix(time-picker): remove extra css reset (#6483) ([`b08054434`](https://github.com/carbon-design-system/carbon/commit/b08054434))
- fix(button): ensure rem in variable definition level (#6470) ([`ba5afb6ef`](https://github.com/carbon-design-system/carbon/commit/ba5afb6ef))
- fix(SideNav): a11y fixes for SideNavMenu, MenuItem (#6468) ([`efa7a9bb1`](https://github.com/carbon-design-system/carbon/commit/efa7a9bb1))
- fix(theme-tokens): increase button padding to match spec (#6457) ([`6aeb25fe5`](https://github.com/carbon-design-system/carbon/commit/6aeb25fe5))
- fix(tooltip): change default styles (#6451) ([`af9543160`](https://github.com/carbon-design-system/carbon/commit/af9543160))
- fix(layout): adjust z-index values (#6365) ([`c2a8be5cb`](https://github.com/carbon-design-system/carbon/commit/c2a8be5cb))
- fix(form): fix rerender issue with helpertext on safari (#6394) ([`066cec250`](https://github.com/carbon-design-system/carbon/commit/066cec250))
- fix(InlineLoading): update success svg color (#6359) ([`082b7e4a0`](https://github.com/carbon-design-system/carbon/commit/082b7e4a0))
- fix(dropdown): add `list-box` import as dependency (#6414) ([`83d0df577`](https://github.com/carbon-design-system/carbon/commit/83d0df577))
- fix(inline-dropdown): set min width to prevent shrinkage (#6297) ([`2c9c98108`](https://github.com/carbon-design-system/carbon/commit/2c9c98108))
- fix(data-table): sortable, boolean column, and overflow menu row fixes (#6268) ([`5d7986990`](https://github.com/carbon-design-system/carbon/commit/5d7986990))
- feat(CodeSnippet): allow copy button to be optional (#6505) ([`ad4d9a298`](https://github.com/carbon-design-system/carbon/commit/ad4d9a298))
- feat(grid): add narrow grid mode (#6520) ([`2c7f9aab5`](https://github.com/carbon-design-system/carbon/commit/2c7f9aab5))
- feat(text-input): add warn prop (#5918) ([`a5e27b74c`](https://github.com/carbon-design-system/carbon/commit/a5e27b74c))

#### @carbon/themes 
- feat(ContentSwitcher): add support for light variant (#6442) ([`c5445a57f`](https://github.com/carbon-design-system/carbon/commit/c5445a57f))

### CODE REVIEW

**Comment by tay1orjones** [comment]
👍  I'm working through this PR to resolve testing failures and complete the upgrade on our end, we have a couple related issues to fix as part of this.

**Comment by tay1orjones** [comment]
@dakahn if you wouldn't mind giving me access to push to your fork I can push the fixes directly to this PR instead of opening a new one.

I've got all test failures resolved except an issue with ComboBox that I'm still working to figure out.

**Comment by dakahn** [comment]
I can also just close this PR and we can reopen another one if that's cleaner :surfer:

**Comment by tay1orjones** [comment]
This is now ready to review while I tie up some loose ends

I need to finish increasing test coverage for ComboBox (it's slightly below 80% after the refactor to remove CarbonComboBox). I also updated with latest from `master` and there's 2 newly failing tests I'll fix.

**Comment by dakahn** [comment]
@tay1orjones Awesome! Latest master is v10.18. Do you mean this includes that release now?

**Comment by tay1orjones** [comment]
@dakahn No, sorry for the confusion - I meant the latest from our repo.

**Comment by tay1orjones** [comment]
Release notes working draft below. I'll update it with notable items as we review this. Once published, I'll edit the release notes in github and copy/paste the markdown, and make note of it in slack so consumers are aware of items they'll have to mitigate.

----
> 
> ## Important items to note when upgrading:
> 
> ### This release updates our Carbon version to v10.17.0
>
> ### Modifications to dropdown listbox rendering
> - This update includes a new version of downshift. All dropdowns (Dropdown, ComboBox, etc) no longer render the ListBox until the list has been opened. This means that any automated tests that assert or interact with listbox items will need to be refactored to first open the dropdown. You can see in the PR how we refactored many of our tests to fix this.
> 
> ### Tooltips that are snapshot tested need an `id`
> - A bug was introduced with the unique IDs of Tooltips. Any tooltip that you snapshot test will need to have an `id` that is identical to `tooltipId`, otherwise every time the snapshot is updated it will have a new value and continually fail. We've opened PR https://github.com/carbon-design-system/carbon/pull/6744 to address this in Carbon.

File: src/utils/__tests__/__snapshots__/publicAPI.test.js.snap [hunk 1]
```diff
@@ -14320,31 +14335,15 @@ Map {
...
-            "breakingChanges": Object {
-              "args": Array [
-                Object {
-                  "resetInputOnSelection": Object {
-                    "type": "bool",
-                  },
-                },
-              ],
-              "type": "shape",
-            },
```
Comment (tay1orjones) [inline src/utils/__tests__/__snapshots__/publicAPI.test.js.snap:14332]: Change comes from Carbon, they've mitigated via a function that maps the old prop names to the new ones.

File: src/utils/__tests__/__snapshots__/publicAPI.test.js.snap [hunk 2]
```diff
@@ -14320,31 +14335,15 @@ Map {
...
             "children": Object {
               "type": "func",
             },
             "defaultHighlightedIndex": Object {
               "type": "number",
             },
-            "defaultInputValue": Object {
-              "type": "string",
-            },
```
Comment (tay1orjones) [inline src/utils/__tests__/__snapshots__/publicAPI.test.js.snap:14341]: Change comes from Carbon, they've mitigated via a function that maps the old prop names to the new ones.

File: src/utils/__tests__/__snapshots__/publicAPI.test.js.snap [hunk 3]
```diff
@@ -14320,31 +14335,15 @@ Map {
...
             "defaultIsOpen": Object {
               "type": "bool",
             },
-            "defaultSelectedItem": Object {
-              "type": "any",
-            },
```
Comment (tay1orjones) [inline src/utils/__tests__/__snapshots__/publicAPI.test.js.snap:14347]: Change comes from Carbon, they've mitigated via a function that maps the old prop names to the new ones.

File: src/utils/__tests__/__snapshots__/publicAPI.test.js.snap [hunk 4]
```diff
@@ -14425,17 +14448,20 @@ Map {
...
           },
         ],
         "type": "shape",
       },
       "editOptionText": Object {
         "type": "string",
       },
-      "findHighlightedIndex": Object {
-        "type": "func",
-      },
```
Comment (tay1orjones) [inline src/utils/__tests__/__snapshots__/publicAPI.test.js.snap:14437]: Was previously used when we copied ComboBox over to CarbonComboBox. Refactored ComboBox no longer needs this override. CarbonComboBox was removed, we now use the stock component from Carbon.

File: src/utils/__tests__/__snapshots__/publicAPI.test.js.snap [hunk 5]
```diff
@@ -14479,9 +14505,6 @@ Map {
         "isRequired": true,
         "type": "func",
       },
-      "onHighligtedIndexChange": Object {
-        "type": "func",
-      },
```
Comment (tay1orjones) [inline src/utils/__tests__/__snapshots__/publicAPI.test.js.snap:14484]: Was previously used when we copied ComboBox over to CarbonComboBox. Refactored ComboBox no longer needs this override. CarbonComboBox was removed, we now use the stock component from Carbon.

File: src/utils/__tests__/__snapshots__/publicAPI.test.js.snap [hunk 6]
```diff
@@ -15468,14 +15550,17 @@ Map {
...
+        },
         "persistant": [Function],
         "persistent": Object {
           "type": "bool",
         },
         "placeHolderText": Object {
           "type": "string",
         },
-        "searchContainerClasses": Object {
+        "searchContainerClass": Object {
```
Comment (tay1orjones) [inline src/utils/__tests__/__snapshots__/publicAPI.test.js.snap:15563]: Change from Carbon. The old prop name should still work.

File: src/components/Breadcrumb/Breadcrumb.jsx [hunk 7]
```diff
@@ -115,7 +115,6 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
                     {...child.props}
                     title={child.props.children}
                     key={`${child.props.children}-${i}`}
-                    primaryFocus={i === 0}
```
Comment (sstone2423) [inline src/components/Breadcrumb/Breadcrumb.jsx:118]: What was this for?

File: src/components/Button/__snapshots__/Button.story.storyshot [hunk 8]
```diff
@@ -122,6 +122,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Butto
         aria-hidden="true"
         aria-label="Button icon"
         className="bx--btn__icon"
+        fill="currentColor"
```
Comment (sstone2423) [inline src/components/Button/__snapshots__/Button.story.storyshot:125]: Is this a new carbon variable?

File: src/components/Card/CardRangePicker.jsx [hunk 9]
```diff
@@ -73,7 +73,6 @@ const CardRangePicker = ({
               [`${iotPrefix}--card--overflow-menuitem-active`]:
                 timeRange === '' || isNil(timeRange),
             })}
-            primaryFocus
```
Comment (sstone2423) [inline src/components/Card/CardRangePicker.jsx:76]: Was focus something that got fixed?

File: src/components/Card/__snapshots__/Card.story.storyshot [hunk 10]
```diff
@@ -560,17 +577,6 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Card
                   Cancel
                 </button>
               </div>
-              <div
```
Comment (sstone2423) [inline src/components/Card/__snapshots__/Card.story.storyshot:563]: Is it intentional that this moved? Looks like it moved in front of the cancel button

File: src/components/ComboBox/_combo-box.scss [hunk 11]
```diff
@@ -29,16 +15,49 @@
...
-    .#{$prefix}--list-box__menu-item:first-child {
-      &::after {
-        content: attr(data-text);
-        color: $text-03;
-        position: absolute;
-        right: $spacing-03;
-        top: 50%;
-        transform: translateY(-50%);
-      }
+    $ai-apps-combobox-input-height-sm: $spacing-07;
```
Comment (sstone2423) [inline src/components/ComboBox/_combo-box.scss:20]: These are cool. Are they variables in the JS?

File: src/components/Dashboard/__snapshots__/Dashboard.story.storyshot [hunk 12]
```diff
@@ -1260,10 +1262,12 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Dashb
                         open={false}
                         tabIndex={0}
                         title="Select time range"
+                        type="button"
```
Comment (sstone2423) [inline src/components/Dashboard/__snapshots__/Dashboard.story.storyshot:1265]: Not sure if this was added by us, but its redundant

File: src/components/Dashboard/__snapshots__/Dashboard.story.storyshot [hunk 13]
```diff
@@ -1568,249 +1579,260 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Dashb
...
+                                </button>
+                              </th>
+                              <th
+                                aria-sort="none"
+                                className="table-header-label-start iot--table-head--table-header table-header-sortable"
+                                scope="col"
                                 style={
                                   Object {
-                                    "--table-header-width": "",
+                                    "width": undefined,
```
Comment (sstone2423) [inline src/components/Dashboard/__snapshots__/Dashboard.story.storyshot:1668]: This seems incorrect. Do we have control over this?

File: src/components/FlyoutMenu/__snapshots__/FlyoutMenu.story.storyshot [hunk 14]
```diff
@@ -72,103 +73,108 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT Exper
...
+              id="flyout-tooltip"
+              onBlur={[Function]}
+              onContextMenu={[Function]}
+              onFocus={[Function]}
+              onMouseOut={[Function]}
+              onMouseOver={[Function]}
+              open={true}
+              role="tooltip"
+              style={
+                Object {
```
Comment (sstone2423) [inline src/components/FlyoutMenu/__snapshots__/FlyoutMenu.story.storyshot:100]: should this be inline?

File: src/components/List/SimpleList/SimpleList.test.jsx [hunk 15]
```diff
@@ -11,7 +11,7 @@ const getFatRowListItems = num =>
   Array(num)
     .fill(0)
     .map((i, idx) => ({
-      id: idx + 1,
+      id: `${idx + 1}`,
```
Comment (sstone2423) [inline src/components/List/SimpleList/SimpleList.test.jsx:14]: Thanks for this!

File: src/components/SideNav/__snapshots__/SideNav.story.storyshot [hunk 16]
```diff
@@ -164,13 +166,11 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/SideN
             >
               <li
                 className="bx--side-nav__menu-item"
-                role="none"
               >
                 <button
                   className="bx--side-nav__link"
                   label="Yet another link"
                   onClick={[Function]}
-                  role="menuitem"
```
Comment (sstone2423) [inline src/components/SideNav/__snapshots__/SideNav.story.storyshot:173]: I don't remember why this was originally added. We should double check to see if this and the role=none above change anything

File: src/components/SideNav/__snapshots__/SideNav.story.storyshot [hunk 17]
```diff
@@ -225,7 +226,6 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/SideN
           >
             <button
               aria-expanded={false}
-              aria-haspopup="true"
```
Comment (sstone2423) [inline src/components/SideNav/__snapshots__/SideNav.story.storyshot:228]: I'm not sure what this prop does. I'll double check

File: src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx [hunk 18]
```diff
@@ -181,12 +181,14 @@ class RowActionsCell extends React.Component {
                     onOpen={this.handleOpen}
                     onClose={this.handleClose}
                     className={`${iotPrefix}--row-actions-cell--overflow-menu`}
+                    selectorPrimaryFocus={`.${iotPrefix}--action-overflow-item--initialFocus`}
```
Comment (sstone2423) [inline src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx:184]: Whats the difference between this and the `primaryFocus` prop ?

File: src/components/Table/TableViewDropdown/__snapshots__/TableViewDropdown.story.storyshot [hunk 19]
```diff
@@ -136,409 +108,17 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Table
...
-            aria-labelledby="iot--view-dropdown"
+            aria-labelledby="downshift-13-label"
             className="bx--list-box__menu"
-            id="iot--view-dropdown__menu"
+            id="downshift-13-menu"
+            onBlur={[Function]}
+            onKeyDown={[Function]}
+            onMouseLeave={[Function]}
             role="listbox"
-          >
```
Comment (sstone2423) [inline src/components/Table/TableViewDropdown/__snapshots__/TableViewDropdown.story.storyshot:145]: Do you know why this massive chunk got deleted?

File: src/components/SideNav/__snapshots__/SideNav.story.storyshot (hunk 16 above)
Comment (sstone2423) [reply src/components/SideNav/__snapshots__/SideNav.story.storyshot:173]: It looks like it helps tell the accessibility let it know to shift focus to this item when you tab into the list

File: src/components/SideNav/__snapshots__/SideNav.story.storyshot (hunk 17 above)
Comment (sstone2423) [reply src/components/SideNav/__snapshots__/SideNav.story.storyshot:228]: I think we do still want this: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-links.html

File: src/components/Breadcrumb/Breadcrumb.jsx (hunk 7 above)
Comment (davidicus) [reply src/components/Breadcrumb/Breadcrumb.jsx:118]: This is the old way to set primary focus element. Is now deprecated and Taylor added the `selectorPrimaryFocus` to the OverflowMenu

File: src/components/Card/CardRangePicker.jsx (hunk 9 above)
Comment (davidicus) [reply src/components/Card/CardRangePicker.jsx:76]: This should also have a `selectorPrimaryFocus` for the `OverflowMenu`

File: src/components/Card/CardToolbar.jsx [hunk 20]
```diff
@@ -100,7 +100,6 @@ const CardToolbar = ({
                 onCardAction(CARD_ACTIONS.EDIT_CARD);
               }}
               itemText={i18n.editCardLabel}
-              primaryFocus
```
Comment (davidicus) [inline src/components/Card/CardToolbar.jsx:103]: If we remove this we also need to add `selectorPrimaryFocus` for `OverflowMenu`

File: src/components/FilterTags/FilterTags.jsx [hunk 21]
```diff
@@ -103,7 +103,6 @@ const FilterTags = ({ children, hasOverflow, id, tagContainer }) => {
         >
           {overflowItems.map((child, i) => (
             <OverflowMenuItem
-              primaryFocus={i === 0}
```
Comment (davidicus) [inline src/components/FilterTags/FilterTags.jsx:106]: Same as above. We need to add a `selectorPrimaryFocus` prop to `OverlowMenu`

File: src/components/List/ListItem/ListItem.story.jsx [hunk 22]
```diff
@@ -36,7 +36,7 @@ storiesOf('Watson IoT Experimental/ListItem', module)
         : rowActionSet === 'multi'
         ? [
             <OverflowMenu flipped>
-              <OverflowMenuItem itemText="Edit" primaryFocus />
+              <OverflowMenuItem itemText="Edit" />
```
Comment (davidicus) [inline src/components/List/ListItem/ListItem.story.jsx:39]: We need to add a `selectorPrimaryFocus` prop to `OverlowMenu`

File: src/components/OverflowMenu/OverflowMenu.story.jsx [hunk 23]
```diff
@@ -45,7 +45,7 @@ const props = {
 const OverflowMenuExample = ({ overflowMenuProps, overflowMenuItemProps }) => (
   <>
     <OverflowMenu {...overflowMenuProps}>
-      <OverflowMenuItem {...overflowMenuItemProps} itemText="Option 1" primaryFocus />
```
Comment (davidicus) [inline src/components/OverflowMenu/OverflowMenu.story.jsx:48]: We need to add a `selectorPrimaryFocus` prop to `OverlowMenu`

File: src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx (hunk 18 above)
Comment (davidicus) [reply src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx:184]: This is the new way Carbon is doing focus for the OverflowMenu

File: src/components/TableCard/TableCard.jsx [hunk 24]
```diff
@@ -367,7 +367,6 @@ const TableCard = ({
                   actionId: item.id,
                 });
               }}
-              primaryFocus={i === 0}
```
Comment (davidicus) [inline src/components/TableCard/TableCard.jsx:370]: We need to add a `selectorPrimaryFocus` prop to `OverlowMenu`

File: src/components/Breadcrumb/Breadcrumb.jsx (hunk 7 above)
Comment (tay1orjones) [reply src/components/Breadcrumb/Breadcrumb.jsx:118]: Nope. The first item is focused by default now when the menu opens

File: src/components/Card/CardRangePicker.jsx (hunk 9 above)
Comment (tay1orjones) [reply src/components/Card/CardRangePicker.jsx:76]: If it was on the first item it's no longer needed. The menu auto focuses the first element now.

File: src/components/ComboBox/_combo-box.scss (hunk 11 above)
Comment (tay1orjones) [reply src/components/ComboBox/_combo-box.scss:20]: No, just something I made in the sass so it's a bit easier to read and understand what I'm doing

File: src/components/FilterTags/FilterTags.jsx (hunk 21 above)
Comment (davidicus) [reply src/components/FilterTags/FilterTags.jsx:106]: Carbon provides a  default for first child

File: src/components/Button/__snapshots__/Button.story.storyshot (hunk 8 above)
Comment (tay1orjones) [reply src/components/Button/__snapshots__/Button.story.storyshot:125]: Nope! https://css-tricks.com/currentcolor/

File: src/components/SideNav/__snapshots__/SideNav.story.storyshot (hunk 16 above)
Comment (tay1orjones) [reply src/components/SideNav/__snapshots__/SideNav.story.storyshot:173]: Yep, to my knowledge this was an intentional fix from Carbon. Neither are something we set.

File: src/components/SideNav/__snapshots__/SideNav.story.storyshot (hunk 17 above)
Comment (tay1orjones) [reply src/components/SideNav/__snapshots__/SideNav.story.storyshot:228]: This was intentional - see the comments here https://github.com/carbon-design-system/carbon/issues/4944#issuecomment-658344895

File: src/components/Table/TableViewDropdown/__snapshots__/TableViewDropdown.story.storyshot (hunk 19 above)
Comment (tay1orjones) [reply src/components/Table/TableViewDropdown/__snapshots__/TableViewDropdown.story.storyshot:145]: Yep, the list box is no longer rendered to the DOM until the list box has been actually opened. From my understanding it's an a11y improvement that is tied into the new version of Downshift that Carbon upgraded to.

File: src/components/FlyoutMenu/__snapshots__/FlyoutMenu.story.storyshot (hunk 14 above)
Comment (tay1orjones) [reply src/components/FlyoutMenu/__snapshots__/FlyoutMenu.story.storyshot:100]: Yep, it's how Carbon does the tooltip positioning/offset

File: src/components/Card/__snapshots__/Card.story.storyshot (hunk 10 above)
Comment (tay1orjones) [reply src/components/Card/__snapshots__/Card.story.storyshot:563]: Yep https://github.com/carbon-design-system/carbon/pull/6566

Though I've uncovered a bug with batch actions thanks to looking into this. Will push a fix.

File: src/components/Dashboard/__snapshots__/Dashboard.story.storyshot (hunk 12 above)
Comment (tay1orjones) [reply src/components/Dashboard/__snapshots__/Dashboard.story.storyshot:1265]: Intentional change from Carbon: https://github.com/carbon-design-system/carbon/pull/6612

File: src/components/Dashboard/__snapshots__/Dashboard.story.storyshot (hunk 13 above)
Comment (tay1orjones) [reply src/components/Dashboard/__snapshots__/Dashboard.story.storyshot:1668]: I'm not sure, I'll look into it

**Comment by tay1orjones** [comment]
@davidicus focus issue has been fixed and I believe all other comments addressed, please re review

@sstone2423 I've responded to your comments - I think we're good to go pending your re-review.

File: src/components/Dashboard/__snapshots__/Dashboard.story.storyshot (hunk 13 above)
Comment (tay1orjones) [reply src/components/Dashboard/__snapshots__/Dashboard.story.storyshot:1668]: Update: The diff is very hard to read here, but if you look above you'll see this is the same as before. Also below you'll see the --table-header-width is still given to the button. I think there was just a small change in DOM structure and it caused the diff to go wild.

**Pull request merged**